LICENSE

```diff
@@ -0,0 +1,19 @@
+The MIT License (MIT)
```

Comment:
Can we copy the one from Jaguar so they are identical (modulo year perhaps)?

Reply:
Have used the same as the one from Jaguar now and updated year to 2022